### PR TITLE
Add send_invoice functionality

### PR DESCRIPTION
**Pull request by gmhawash**

### CODE REVIEW

**Comment by minimul** [comment]
Thanks. I'm taking a look now.

File: lib/qbo_api/connection.rb [hunk 1]
```diff
@@ -51,6 +51,7 @@ def raw_request(method, conn:, path:, payload: nil, params: nil)
         when :post, :put
           req.url path
           req.body = payload.to_json
+          req.headers = headers if headers
```
Comment (minimul) [inline lib/qbo_api/connection.rb:54]: Let's move this line to before the start of the case statement so that headers can be set for a `get` call too.

**Comment by minimul** [comment]
Looks great. 

If you could squash the commits into one after making the change I detailed above I'll get this feature merged and released.

**Review by gmhawash — COMMENTED** [review]
I refactored this to allow for sending to any of the entities supported by QBO.  See https://developer.intuit.com/app/developer/search?keywords=send.

I also added minorversion=63 which appears to be required for most of these to work for adding the `sendTo` param.

I will update the spec and squash into one commit after you take a look at this.

**Comment by minimul** [comment]
Well done. Go ahead and squash it up.

**Review by gmhawash — COMMENTED** [review]
I added specs and squashed into one commit.

**Comment by minimul** [comment]
Great job. Merged and released in v. 2.0.2

**Comment by gmhawash** [comment]
Thanks, glad to do it.
Maher

**Pull request merged**